### PR TITLE
feat(bundler sidecar): init docker compose for a standalone bundler sidecar PE-6105

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

**Review by djwhitt — COMMENTED** [review]
@fedellen Did you consider using docker-compose profiles for this? It seems attractive to be able to fire up the entire stack with one command.

File: docker-compose.override.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+  core:
+    environment:
+      ANS104_UNBUNDLE_FILTER: '{ "attributes": { "owner_address": "$BUNDLER_ARWEAVE_ADDRESS" }}'
+      ANS104_INDEX_FILTER: '{ "always": true }'
+    profiles: ['bundler']
```
Comment (fedellen) [inline docker-compose.override.yaml:8]: when using bundler profile, we will index bundles from our bundler wallet

**Comment by fedellen** [comment]
> Did you consider using docker-compose profiles for this? It seems attractive to be able to fire up the entire stack with one command.


[0d326a6](https://github.com/ar-io/ar-io-node/pull/117/commits/0d326a6a01fbefa78ecc300d9b9a8a54223c8c11)

😄

File: README.md [hunk 2]
```diff
@@ -77,6 +77,25 @@ docker-compose up --build
...
+```
+
+Then, run docker compose with the `bundler` profile.
+
+```shell
+docker-compose --profile bundler up
+```
+
+Now, the bundler service will be running alongside the ar.io gateway. Your gateway will now accept data items at `https://my-gateway.net/bundler/tx` 🚀
+
```
[the remark this answers is not in the document]
Comment (fedellen) [reply README.md:98]: draft away 😄

File: docker-compose.override.yaml (hunk 1 above)
Comment (karlprieb) [reply docker-compose.override.yaml:8]: I didn't know about that! profiles are really cool!

**Review by djwhitt — APPROVED** [review]

**Pull request merged**